app/components/rdf-input-fields/custom-field-wrapper.js

```diff
@@ -7,17 +7,26 @@ import { task } from 'ember-concurrency';
 import { validationsForFieldWithType } from '@lblod/submission-form-helpers';
 
 import { JSON_API_TYPE } from 'frontend-lmb/utils/constants';
+import { ADRES_CUSTOM_DISPLAY_TYPE } from 'frontend-lmb/utils/well-known-uris';
 
 export default class RdfInputFieldsCustomFieldWrapperComponent extends Component {
   @consume('on-form-update') onFormUpdate;
```

Comment:
see here we're now consuming three things as well! feels like it belongs together